style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
Great job @rozhi-98 ✋

- [ ] As stated in the project requirements. Kindly add a class to this `h1` and use it in your `style.css` file instead of using the `h1` itself.